### PR TITLE
[5주차] transformers / label_smoothing_loss & positional_encoding.html / 박지은

### PR DESCRIPTION
**Pull request by Jieun-Park99**
# What does this PR do?
Added Korean files for week 5 in the `docs/ko/` 
- Change docs folder file which written in English to Korean.
- Translate `transformers / label_smoothing_loss.html`
- Translate `transformers / positional_encoding.html`

# Before submitting
- [x]  This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

- [x] Did you make sure to update the documentation with your changes? Here are the
[documentation guidelines](https://github.com/labmlai/annotated_deep_learning_paper_implementations/tree/master/docs), and
[here are tips on formatting html and css](https://chanrankim.notion.site/2c2967ff842344a78a04f50d8274c802?pvs=4)

- [ X ] Did you write any new necessary tests?

# Who can review?
Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@Bandi120424 , @Jieun-Park99 , @kimdahyeon977, @mlleo , @movie5, @nehcream2od, @rimiyeyo, @russellgeum @Youngmi-Park 

Pseudo Lab, Please review this PR.

### CODE REVIEW

File: docs/ko/transformers/label_smoothing_loss.html [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-    <meta name="twitter:title" content="Label Smoothing Loss"/>
-    <meta name="twitter:description" content="This is an implementation of label smoothing loss, that can be used as an alternative to cross entropy loss for improved accuracy."/>
+    <meta name="twitter:title" content="라벨 스무딩 손실 함수"/>
+    <meta name="twitter:description" content="이것은 정확도 향상을 위한 대안으로 사용할 수 있는 라벨 스무딩 손실함수의 구현입니다. 이것은 모델의 정확도를 개선하기 위해 크로스 엔트로피 손실 대신 사용할 수 있습니다."/>
     <meta name="twitter:site" content="@labmlai"/>
     <meta name="twitter:creator" content="@labmlai"/>
 
     <meta property="og:url" content="https://nn.labml.ai/transformers/label_smoothing_loss.html"/>
-    <meta property="og:title" content="Label Smoothing Loss"/>
+    <meta property="og:title" content="라벨 수무딩 손실 함수"/>
```
Comment (movie5) [inline docs/ko/transformers/label_smoothing_loss.html]: @Jieun-Park99 스무딩인거 같습니다!

File: docs/ko/transformers/label_smoothing_loss.html [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
-    <meta property="og:site_name" content="Label Smoothing Loss"/>
+    <meta property="og:site_name" content="라벨 스무딩 손실 함수"/>
     <meta property="og:type" content="object"/>
-    <meta property="og:title" content="Label Smoothing Loss"/>
-    <meta property="og:description" content="This is an implementation of label smoothing loss, that can be used as an alternative to cross entropy loss for improved accuracy."/>
+    <meta property="og:title" content="라벨 스무딩 손실 함수"/>
+    <meta property="og:description" content="이것은 정확도 향상을 위한 대안으로 사용할 수 있는 라벨 스무딩 손실함수의 구현입니다. 이것은 모델의 정확도를 개선하기 위해 크로스 엔트로피 손실 대신 사용할 수 있습니다."/>
 
-    <title>Label Smoothing Loss</title>
+    <title>라벨 스무딩 손실 함수 (Label Smoothing Loss)</title>
```
Comment (movie5) [inline docs/ko/transformers/label_smoothing_loss.html:23]: @Jieun-Park99  굿

**Review by movie5 — CHANGES_REQUESTED** [review]
확인했습니다

File: docs/ko/transformers/label_smoothing_loss.html (hunk 1 above)
Comment (Jieun-Park99) [reply docs/ko/transformers/label_smoothing_loss.html]: 수정 완료했습니다!

**Review by movie5 — APPROVED** [review]

**Pull request merged**